peachjam/views/gazette.py

```diff
@@ -122,6 +122,10 @@ class GazetteYearView(YearMixin, FilteredDocumentListView):
     navbar_link = "gazettes"
     locality = None
     group_by_date = "month-year"
+    special_facet_options = [
+        {"name": _("Special Issue"), "value": "special"},
```

Comment:
the `_` must be the lazy variant, so that the translation is done at the last minute.

Reply:
is it okay to change it from the import thereby changing it everywhere in the file?

Reply:
I've seen we do it elsewhere so i guess its okay

Reply:
> I've seen we do it elsewhere so i guess its okay

yes it shouldn't hurt